src/EditorFeatures/CSharp/AutomaticCompletion/Sessions/CurlyBraceCompletionSession.cs

```diff
@@ -224,6 +224,19 @@ public override AdjustNewLinesOperation GetAdjustNewLinesOperation(SyntaxToken p
                 return base.GetAdjustNewLinesOperation(previousToken, currentToken, optionSet, nextOperation);
             }
 
+            public override void AddAlignTokensOperations(List<AlignTokensOperation> list, SyntaxNode node, OptionSet optionSet, NextAction<AlignTokensOperation> nextOperation)
```

Comment:
Why do we have to add this explicitly?  Is it because the formatter normally doesn't do it in block indent?